backend/api/build_kfp_server_api_python_package.sh

```diff
@@ -46,7 +46,7 @@ pushd "$(dirname "$0")"
 
 CURRENT_DIR="$(pwd)"
 DIR="$CURRENT_DIR/$API_VERSION/python_http_client"
-swagger_file="$CURRENT_DIR/$API_VERSION/swagger/kfp_api_single_file.swagger.json"
+swagger_file="$CURRENT_DIR/$API_VERSION/swagger/kfp_api_${API_VERSION}_single_file.swagger.json"
```

Comment:
I also don't think we have to have it. I just added this part so the file names are not identical. But since we have the version in the path, it does feel unnecessary.